frontend/src/app/main/data/team.cljs

```diff
@@ -133,6 +133,8 @@
       (let [team-id' (get state :current-team-id)]
         (if (= team-id' team-id)
           (-> state
+              (dissoc :files)
```

Comment:
This change breaks the optimization of show files already loaded on return to dashboard from workspace.